Formula/forge.rb

```diff
@@ -5,6 +5,11 @@ class Forge < Formula
   sha256 "4ed631cfde6a9c0daf786d68b47719edec024928b3e23dbfb2e786e60ab01097"
   license "BSD-3-Clause"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The `3.6.0` version is coming from one of the tags starting with `af` (i.e., `af3.6.0`), whereas the tags for `forge` versions use the `v1.2.3` format. Checking the Git tags is sufficient if we use the standard regex for Git tags like `v1.2.3`, as this also gives us the correct latest version (`1.0.5`).